### ISSUE TITLE
VMware Horizon

### ISSUE THREAD

**Issue by rapgro** [theforeman/foreman-ansible-modules#1356]
##### SUMMARY
Hello. Are there any chances to get support for (proprietary) VMware Horizon? Horizon is a special product from VMware company to support Blast protocol with better compression for graphics application. It can use composer or instant clones. There's now a powerful RESTful API based on Python to instruct server functionality.

See https://github.com/Magneet/Horizon_Rest_API_Python
https://techzone.vmware.com/resource/using-vmware-horizon-server-rest-api

##### ISSUE TYPE
 - Feature Idea

**Comment by evgeni** [comment]
To get support where? These Ansible modules just talk to the Foreman API. Does Foreman support Horizon?

**Comment by rapgro** [comment]
not supported yet, where to file feature request there?

**Comment by evgeni** [comment]
https://projects.theforeman.org/

**Comment by rapgro** [comment]
yeah, created an account there in community but I fail to find option for new thread

**Comment by evgeni** [comment]
https://projects.theforeman.org/projects/foreman/issues has a "new issue" link in the right upper corner https://projects.theforeman.org/projects/foreman/issues/new

**Comment by rapgro** [comment]
thanks, still waiting for e-mail confirmation
